src/Campaigns/resources/admin/components/CampaignFormModal/index.tsx

```diff
@@ -22,6 +17,7 @@ import {
 import {getGiveCampaignsListTableWindowData} from '../CampaignsListTable';
 import {amountFormatter} from '@givewp/campaigns/utils';
 import TextareaControl from '../CampaignDetailsPage/Components/TextareaControl';
+import {getGoalInputAttributes} from '../../constants/goalInputAttributes';
```

Comment:
I'm not seeing any other exports in this file so maybe we should just export default this function and rename the file to `getGoalInputAttributes` or even `getGoalType`.  

I did something similar for the [SubscriptionPeriod](https://github.com/impress-org/givewp/blob/f5138cbc6bcbed67d20d0474cadb077e097f5b15/src/DonationForms/resources/registrars/templates/groups/DonationAmount/subscriptionPeriod.ts#L58) that is likely way more overkill than we need here but I had fun for it so here you go for [future inspiration](https://github.com/impress-org/givewp/blob/78561c1612fdd7ab74af6931ac42e793122b632e/src/DonationForms/resources/registrars/templates/elements/DonationSummary.tsx#L35) 😄

Reply:
Please, let me know in case I went too far on this: ef389b4756ca6a263112c912fa158c6caf83ba2b

Reply:
It was worth it 😄